SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialWorkerConnection.cpp

```diff
@@ -267,6 +273,12 @@ void USpatialWorkerConnection::ConnectToLocator()
 	ConnectionParams.network.use_external_ip = LocatorConfig.UseExternalIp;
 	ConnectionParams.network.tcp.multiplex_level = LocatorConfig.TcpMultiplexLevel;
 
+	// We want the bridge to worker messages to be compressed; not the worker to bridge messages.
+	// Worker SDK 14.1.0 has a bug where upstream and downstream compression are swapped so we set the upstream settings to use compression.
```

Comment:
Add a TODO to switch these round when the SDK version is updated? If there is a ticket for actually doing the upgrade, adding an AC to it that this is updated should probably be part of it.

Reply:
Done